frontend/src/Settings/Indexers/Indexers/EditIndexerModalContent.js

```diff
@@ -153,6 +154,23 @@ function EditIndexerModalContent(props) {
                 />
               </FormGroup>
 
+              <FormGroup
+                advancedSettings={advancedSettings}
+                isAdvanced={true}
+              >
+                <FormLabel>Maximum Single Episode Age</FormLabel>
+
+                <FormInputGroup
+                  type={inputTypes.NUMBER}
+                  name="seasonSearchMaximumSingleEpisodeAge"
+                  helpText="During a full season search, only season packs will be allowed when the season's most recent episode is older than this setting. Standard series only. Use 0 to disable."
```

Comment:
Reading this again I think the `,` after `During a full season search` is incorrect.

Reply:
One other thing here would be to change `season's most recent episode` to `season's last episode` since this doesn't apply if the whole season hasn't aired.